Sources/Private/Experimental/Animations/VisibilityAnimation.swift

```diff
@@ -0,0 +1,37 @@
+// Created by Cal Stephens on 12/21/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import QuartzCore
+
+extension CALayer {
+  /// Adds an animation for the given `inTime` and `outTime` to this `CALayer`
+  func addVisibilityAnimation(
+    inFrame: AnimationFrameTime,
+    outFrame: AnimationFrameTime,
+    context: LayerAnimationContext)
+  {
+    let animation = CAKeyframeAnimation(keyPath: "hidden")
```

Comment:
I'm pretty sure that `#keyPath(CALayer.isHidden)` resolves to "hidden" if you want to use it here
```suggestion
    let animation = CAKeyframeAnimation(keyPath: #keyPath(CALayer.isHidden))
```

Reply:
You're right! Good thinking

```
(lldb) p #keyPath(CALayer.isHidden)
(String) $R1 = "hidden"
```